source/content/guides/autopilot/05-troubleshoot.md

```diff
@@ -336,7 +336,7 @@ Unfreeze the site in the old dashboard before re-queueing updates. Note, unfreez
 
 ### Diagnosis
 
-Site is running a framework that is not Drupal, Drupal 8, or WordPress. Currently, WordPess Multisite (`wordpress_network`) is not supported.
+Site is running a framework that is not Drupal, Drupal 8, or WordPress. WordPess Multisite (`wordpress_network`) is not supported.
```

Comment:
Can we revert this change? We plan to support Multisite in the future so "currently" is accurate.

Reply:
Done!